src/pages/studyView/TableUtils.tsx

```diff
@@ -47,27 +86,49 @@ export function getCancerGeneFilterToggleIcon(isFilteredByCancerGeneList:boolean
     return <span data-test='cancer-gene-filter' className={classnames(styles.cancerGeneIcon, styles.displayFlex)} style={{color: isFilteredByCancerGeneList ? ICON_FILTER_ON : ICON_FILTER_OFF}}><i className='fa fa-filter'></i></span>;
 }
 
-export function getFreqColumnRender(type: 'mutation' | 'cna', numberOfSamplesProfiled: number, numberOfAlteredCases: number, matchingGenePanelIds: string[], toggleModal: (panelName: string) => void, style?:CSSProperties) {
+type FreqColumnType = 'mutation' | 'fusion' | 'cna';
+
+export function getFreqColumnRender(type: FreqColumnType, numberOfSamplesProfiled: number, numberOfAlteredCases: number, matchingGenePanelIds: string[], toggleModal?: (panelName: string) => void, style?: CSSProperties) {
+    const detailedTypeInfo = type === 'mutation' ? 'mutations' : (type === 'cna' ? 'copy number alterations' : 'fusions')
     const addTotalProfiledOverlay = () => (
         <span style={{display: 'flex', flexDirection: 'column'}} data-test='freq-cell-tooltip'>
-            <span>{`# of samples profiled for ${type === 'mutation' ? 'mutations' : 'copy number alterations'} in this gene: ${numberOfSamplesProfiled.toLocaleString()}`}</span>
+            <span>{`# of samples profiled for ${detailedTypeInfo} in this gene: ${numberOfSamplesProfiled.toLocaleString()}`}</span>
             <GenePanelList
                 genePanelIds={matchingGenePanelIds}
-                toggleModal={toggleModal}
+                toggleModal={toggleModal!}
             />
         </span>
     );
+
+    function getCellContent() {
+        return <span data-test='freq-cell' style={style}>
+                {getFrequencyStr(
+                    (numberOfAlteredCases / numberOfSamplesProfiled) * 100
+                )}
+            </span>;
+    }
+
     return (
         <DefaultTooltip
             placement="right"
             overlay={addTotalProfiledOverlay}
             destroyTooltipOnHide={true}
         >
-            <span data-test='freq-cell' style={style}>
-                {getFrequencyStr(
-                    (numberOfAlteredCases / numberOfSamplesProfiled) * 100
-                )}
-            </span>
+            {getCellContent()}
         </DefaultTooltip>
     );
 }
+
+export function rowIsChecked(entrezGeneId: number, preSelectedRows: AlteredGenesTableUserSelectionWithIndex[], selectedRows: AlteredGenesTableUserSelectionWithIndex[]) {
+    return !!_.find(
```

Comment:
super minor but i thinke _.some would be more semantic.  it returns boolean so wouldn't require the !!

Reply:
Good point.